releases/release-1.31/release-notes/maps/pr-124221-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124221
+releasenote:
+  text: Fixed a typo in the help text for the pod_scheduling_sli_duration_seconds
+    metric in kube-scheduler
```

```suggestion
    metric in kube-scheduler.
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653203528